.travis.yml

```diff
@@ -0,0 +1,11 @@
+language: python
+python:
+- "2.7"
+- "3.4"
+- "3.5"
+install:
```

Comment:
For the functional tests to work, we need to install NodeJS and PhantomJS

```
curl -sL https://deb.nodesource.com/setup_6.x | sudo -E bash -
sudo apt-get install -y nodejs
sudo npm -g install phantomjs-prebuilt
```

Maybe there is no need for `sudo`. Not sure.